config/default_jobs/test_jobs.yaml

```diff
@@ -0,0 +1,26 @@
+# Jobs with a good mix of situations used for testing
+aasysjur:
+  table_name: iasworld.aasysjur
+  min_year: null
+  max_year: null
+  cur: null
+  predicates_path: null
+addn:
+  table_name: iasworld.addn
+  min_year: 2023
```

Comment:
This `min_year` attribute used to reference the `min_year` anchor set in a `define` block at the top of `default_jobs.yaml`, but I've removed that alias because this is the only table that references the anchor in this batch. Here's what the code looked like prior to this diff, for the sake of comparison:

https://github.com/ccao-data/service-spark-iasworld/blob/3fe41bc7df3360794cca6d4a932745c7f0870280/config/default_jobs.yaml#L1-L2

https://github.com/ccao-data/service-spark-iasworld/blob/3fe41bc7df3360794cca6d4a932745c7f0870280/config/default_jobs.yaml#L274-L276